src/encoded/tests/test_search_views.py

```diff
@@ -0,0 +1,522 @@
+    assert r.json['matrix']['y']['group_by'] == [
+        'biosample_ontology.classification',
+        'biosample_ontology.term_name'
+    ]
+    assert 'buckets' in r.json['matrix']['y']['biosample_ontology.classification']
+    assert 'key' in r.json['matrix']['y']['biosample_ontology.classification']['buckets'][0]
+    assert 'biosample_ontology.term_name' in r.json['matrix']['y']['biosample_ontology.classification']['buckets'][0]
+    assert 'search_base' in r.json
+    assert r.json['search_base'] == '/search/?type=Experiment'
+
+
+def test_matrix_response_with_search_term_type_only_clear_filters(workbook, testapp):
+    r = testapp.get('/matrix/?type=Experiment&searchTerm=A549')
+    assert 'clear_filters' in r.json
+    assert r.json['clear_filters'] == '/matrix/?type=Experiment'
+
+
+def test_matrix_response_debug(workbook, testapp):
+    r = testapp.get('/matrix/?type=Experiment&debug=true')
+    assert 'debug' in r.json
+
+
+def test_matrix_response_no_results(workbook, testapp):
+    r = testapp.get(
+        '/matrix/?type=Experiment&status=no_status',
+        status=404
+    )
+    assert r.json['notification'] == 'No results found'
+
+
+def test_summary_response(workbook, testapp):
+    r = testapp.get('/summary/?type=Experiment')
+    assert 'aggregations' not in r.json
+    assert 'facets' in r.json
+    assert 'total' in r.json
+    assert r.json['title'] == 'Summary'
+    assert r.json['@type'] == ['Summary']
+    assert r.json['clear_filters'] == '/summary/?type=Experiment'
+    assert r.json['filters'] == [{'term': 'Experiment', 'remove': '/summary/', 'field': 'type'}]
+    assert r.json['@id'] == '/summary/?type=Experiment'
+    assert r.json['total'] >= 22
+    assert r.json['notification'] == 'Success'
+    assert r.json['title'] == 'Summary'
+    assert 'facets' in r.json
+    assert r.json['@context'] == '/terms/'
+    assert 'matrix' in r.json
+    assert r.json['matrix']['x']['group_by'] == 'status'
+    assert r.json['matrix']['y']['group_by'] == ['replication_type']
+    assert 'buckets' in r.json['matrix']['y']['replication_type']
+    assert 'key' in r.json['matrix']['y']['replication_type']['buckets'][0]
+    assert 'status' in r.json['matrix']['y']['replication_type']['buckets'][0]
+    assert 'search_base' in r.json
+    assert r.json['search_base'] == '/search/?type=Experiment'
+
+
+def test_collection_listing_es_view(workbook, testapp):
+    r = testapp.get(
+        '/experiments/'
+    )
+    assert '@graph' in r.json
+    assert '@id' in r.json
+    assert 'facets' in r.json
+    assert 'filters' in r.json
+    assert 'all' in r.json
+    assert 'columns' in r.json
+    assert 'clear_filters' in r.json
+    assert r.json['clear_filters'] == '/search/?type=Experiment'
+    assert r.json['@type'] == ['ExperimentCollection', 'Collection']
+    assert r.json['@context'] == '/terms/'
+
+
+def test_collection_listing_es_view_item(workbook, testapp):
+    r = testapp.get(
+        '/Experiment'
```

Comment:
this is an odd url.  Isn't the collection /experiments/?   /Experiment redirects.

Reply:
This is testing the redirect.